run.sh

```diff
@@ -14,6 +14,11 @@ export HOME=/var/lib/logstash
 
 : ${OUTPUT_ELASTICSEARCH:=true}
 : ${ELASTICSEARCH_HOST:=127.0.0.1:9200}
+: ${ELASTICSEARCH_SSL_ENABLED:=false}
+: ${ELASTICSEARCH_CERTIFICATE_VERIFICATION:=false}
+: ${ELASTICSEARCH_USER:="none"}
```

Comment:
If left empty strings it'll break template validation. We pass those in deployment so this is only  a placeholder, but I think I change that.